Riot/Modules/Onboarding/AuthenticationCoordinator.swift

```diff
@@ -613,6 +613,10 @@ extension AuthenticationCoordinator: AuthenticationServiceDelegate {
         
         return true
     }
+
+    func authenticationService(_ service: AuthenticationService, didUpdateStateWithLink link: UniversalLink) {
+        //  updating current views according to the link doesn't seem easy.
```

Comment:
I think the correct route here would be to call `callback?(.cancel(.register))` for a registration link and `callback?(.cancel(.login))` for the homeserver config link. Maybe with an indicator saying that configuration was successful. What do you think?

Reply:
Makes sense, updated.

Reply:
`Success` was not present in the strings, `Done` used instead.